src/components/settings/SpendingLimits/NewSpendingLimit/steps/ReviewSpendingLimit.tsx

```diff
@@ -141,22 +143,34 @@ export const ReviewSpendingLimit = ({ data, onSubmit }: Props) => {
         <EthHashInfo address={data.beneficiary} shortAddress={false} hasExplorer showCopyButton />
       </Box>
 
-      <Typography color={({ palette }) => palette.text.secondary}>Reset Time</Typography>
+      <Typography color={({ palette }) => palette.text.secondary}>Reset time</Typography>
       {existingSpendingLimit ? (
         <>
-          <Box display="flex" alignItems="center" gap="4px" mb={2}>
-            <Typography color="error" sx={{ textDecoration: 'line-through' }}>
-              {relativeTime(existingSpendingLimit.lastResetMin, existingSpendingLimit.resetTimeMin)}
-            </Typography>
-            {' →'}
-            <Typography>{resetTime}</Typography>
-          </Box>
+          <SpendingLimitLabel
+            label={
+              <>
+                <Typography color="error" sx={{ textDecoration: 'line-through' }} display="inline" component="span">
+                  {relativeTime(existingSpendingLimit.lastResetMin, existingSpendingLimit.resetTimeMin)}
+                </Typography>
+                {' →'}
```

Comment:
Since you're touching this, can we also compare that the reset time is actually being changed and not show the "<s>one time</s> →" if not?